### DISCUSSION TITLE
Upload firmware and code with pybricksdev

### DISCUSSION THREAD

**Discussion by Gcopper1984** [pybricks/support#546] [Q&A] [answered]
Hello to everyone,

   is it possibile using pybricksdev (or something else) to upload a firmware and a custom program just like the online editor do?

Thank you :-)

**Comment by laurensvalk** [comment, marked as answer]
Yes, just modify the main.py program in the firmware zip file.

**Reply by Gcopper1984** [reply]
Ok, thank you!
